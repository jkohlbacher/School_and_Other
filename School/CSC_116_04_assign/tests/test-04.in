ourm
desa
allo
ball
haus
inne
fici
ntwy
