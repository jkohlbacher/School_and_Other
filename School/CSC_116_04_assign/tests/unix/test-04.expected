Read 178691 words.
Tiles: [ourm]
Solution: 1 [mor] Score [5]
Solution: 2 [rom] Score [5]
Solution: 3 [rum] Score [5]
Tiles: [desa]
Solution: 1 [sade] Score [5]
Tiles: [allo]
Solution: 1 [olla] Score [4]
Tiles: [ball]
Solution: 1 [ball] Score [6]
Tiles: [haus]
Solution: 1 [ahs] Score [6]
Solution: 2 [ash] Score [6]
Solution: 3 [has] Score [6]
Solution: 4 [sha] Score [6]
Tiles: [inne]
Solution: 1 [nine] Score [4]
Tiles: [fici]
Solution: 1 [if] Score [5]
Tiles: [ntwy]
Solution: 1 [wyn] Score [9]
